wow.js

```diff
@@ -0,0 +1,29 @@
+const { KernelMessage, Kernel, KernelManager, ServerConnection, ContentsManager } = require('@jupyterlab/services');
```

Comment:
I have no idea where this file came from? Running tests?

Reply:
I added this. Please remote this.